core/spec/models/spree/order_spec.rb

```diff
@@ -94,7 +94,7 @@
       let!(:store_credit_payment_method) { create(:store_credit_payment_method) }
       let(:order_total) { 500.00 }
       let(:store_credit) { create(:store_credit, amount: order_total) }
-      let(:order) { create(:order_with_line_items, user: store_credit.user, line_items_price: order_total) }
+      let(:order) { create(:order_with_line_items, state: :complete, user: store_credit.user, line_items_price: order_total) }
```

Comment:
What about using the [`order_ready_to_complete`](https://github.com/solidusio/solidus/blob/dab9035e75e76556bb0dc2cbcbf666fed9427d6c/core/lib/spree/testing_support/factories/order_factory.rb#L83-L100) factory and call `complete` in the before block?

Reply:
Changed, and works as well!